src/writeData/subscriptions/components/BrokerForm.scss

```diff
@@ -0,0 +1,68 @@
+.create-broker-form {
```

Comment:
I still need to go through my scss files and use variables for color and sizing where it’s possible

Reply:
done